index.js

```diff
@@ -173,8 +187,6 @@ Parser.prototype._flush = function(done) {
         translationsHash = helpers.hashFromString( key, self.keySeparator, translationsHash );
     }
 
-
```

Comment:
Do not remove lines, respect the style of the original author. The 3 spaces is consistent throughout the project.